src/js_object_accessor.hpp

```diff
@@ -181,6 +181,7 @@ class NativeAccessor {
         return SetClass<JSEngine>::create_instance(m_ctx, std::move(set));
     }
     ValueType box(realm::object_store::Dictionary dictionary) {
+        throw std::runtime_error("Dictionaries are not yet supported");
         return dictionary_adapter.wrap(m_ctx, dictionary);
```

Comment:
I'd love to see this line and 279 deleted. That may enable you to delete 28 as well.